be/src/runtime/load_channel.cpp

```diff
@@ -123,7 +123,10 @@ void LoadChannel::add_chunk(const PTabletWriterAddChunkRequest& request, PTablet
         auto& pchunk = request.chunk();
         RETURN_RESPONSE_IF_ERROR(_build_chunk_meta(pchunk), response);
         RETURN_RESPONSE_IF_ERROR(_deserialize_chunk(pchunk, chunk, &uncompressed_buffer), response);
+        auto chunk_mem_usage = chunk.memory_usage();
```

Comment:
This can lead to over-memory statistics. If the current problem is not that serious, I will think about how to do later.

Reply:
The memory has been counted into the process memory statistics during desrialize_chunk through hook.

Reply:
This memory is better accounted for independently. For example, MemTable takes up very little memory, and we can't make the Queue very long, because it doesn't make much sense.